src/App.tsx

```diff
@@ -1,20 +1,40 @@
-import React from "react";
+import React, { useEffect} from "react";
 import { Routes, Route } from "react-router-dom";
 import Home from "./pages/home/Home";
 import Profile from "./pages/profile/Profile";
 import Feeds from "./pages/feed/Feeds";
-import Navbar from "./shared/navbar/Navbar";
+import { Signup } from "./pages/authentication/signup/Signup";
+import { Login } from "./pages/authentication/login/Login";
```

Comment:
Barrel import

Reply:
I will move those to one file when I will do refactoring